test/TestHelper.cpp

```diff
@@ -180,8 +236,16 @@ void ImportTest::exportTest(bytes const& _output, State const& _statePost)
 	// export logs
 	m_TestObject["logs"] = exportLog(_statePost.pending().size() ? _statePost.log(0) : LogEntries());
 
-	// export post state
+	// compare expected state with post state
+	if (m_TestObject.count("expect") > 0)
```

Comment:
Please make commandline flag for that, or only give a wanring if this fails. Because we may have protocol changes, which will give a different post state (most certainly). So it should be possible to check onkly single value in the "expect" onject. For example only that the value at key 0 is 1  (for one address). But dont compare the complete post state.